pkg/server/response.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+//Error returns an error if request's not valid
+func Error(w http.ResponseWriter, code int, message string) {
+	JSON(w, code, map[string]string{"error": message})
```

Comment:
With http status codes it is clear you have an error, maybe, `reason` is better here